packages/vuetify/src/components/VSlider/VSlider.ts

```diff
@@ -283,7 +283,7 @@ export default mixins<options &
         attrs: {
           value: this.internalValue,
           id: this.computedId,
-          disabled: this.isDisabled,
+          disabled: true,
```

Comment:
Value of this input tag in `VSlider` shall be only changed by `this.internalValue`. Disable it always will make sure its value will not be changed by other events.

All other changes in this pull request is html snapshot in unit tests.